### DISCUSSION TITLE
Markdown ordered list behavior inconsistent between Github & Github pages

### DISCUSSION THREAD

**Discussion by wiz0u** [community/community#42087] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

In my repository, I use Github Pages without a specific generator, just plain old Markdown *.MD files.

On the page linked below, under §8, I've noticed a discrepancy in the rendered numbering of markdown ordered lists:
When viewed in Github repository, the lists goes as expected: **1. 2. 3. 4. (intercut) 5. 6. 7....**
For the same page, when viewed in Github pages, the lists is: **1. 2. 3. 4. (intercut) 1. 2. 3....**

In the repository: https://github.com/wiz0u/WTelegramClient/blob/master/FAQ.md#prevent-ban
In Github pages: https://wiz0u.github.io/WTelegramClient/FAQ#prevent-ban

(is this the right place to signal this problem?)
Thanks

**Comment by yoannchaudet** [comment]
Hello,

GitHub Pages and GitHub have nothing in common when it comes to markdown. GitHub Pages actually does not know anything about markdown but you are probably referring to [Jekyll](https://jekyllrb.com/) which is the static site generator used by default.

You may obtain the behavior you are after by changing Jekyll's markdown engine to use GitHub Flavored Markdown. More here: https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/setting-a-markdown-processor-for-your-github-pages-site-using-jekyll.

**Reply by wiz0u** [reply]
thanks, I will look about this
